packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -225,6 +241,33 @@ export function buildRootObject() {
         issuerTable.getIssuerRecordByIssuer(issuer).brand,
       getAmountMath,
       makeZCFMint,
+      addEmptySeat: keyword => {
+        const initialAllocation = harden({ [keyword]: undefined });
+        const proposal = harden({ want: { [keyword]: undefined } });
+        const { notifier, updater } = makeNotifierKit();
+        const zoeSeatAdminPromiseKit = makePromiseKit();
+
+        E(zoeInstanceAdmin)
+          .makeEmptySeat(keyword, initialAllocation, proposal)
+          .then(({ seatAdmin: zoeSeatAdmin, seatData }) => {
```

Comment:
Let's call this the ZoeSeatAdmin in the returned object too. It's called that in Zoe.